### DISCUSSION TITLE
<!-- removed -->

### DISCUSSION THREAD

**Discussion by uqsme** [SeaDve/Kooha#176] [General]

**Comment by lumingzh** [comment]
Yes, you're right. I have no contributions to kooha, but I have contributions to mousai which is under the same project "FOSS Projects". So it's weblate's problem not mine.
Thank you.

**Comment by uqsme** [comment]
> Yes, you're right. I have no contributions to kooha, but I have contributions to mousai which is under the same project "FOSS Projects". So it's weblate's problem not mine. Thank you.

估计是weblate那个语料库的问题，mousai翻译后，相同字符串 `translator-credits` 被同步到了kooha，这就是weblate的坑

**Comment by SeaDve** [comment]
It might be caused by Weblate synchronizing strings across components since Mousai, Kooha, and Breathing (the components) share the same project.

This could be fixed by disabling the sharing between components, but not sure if everyone agrees with that. Personally, I would disable it if that would fix the issue.
